### PR TITLE
[master < T595] Add docs for remove label function on cpp api

### PR DESCRIPTION
**Pull request by ind1xa**
### Description

Added docs for remove label function on cpp api.

### Pull request type

Please delete options that are not relevant and check the ones that are.

- [ ] Documentation fix
- [ ] New documentation page 
- [ ] Base PR for the release
- [x] Documentation improvements
- [ ] Other (please describe):

### Related PRs and issues

PR this doc page is related to: 
Closes (link to issue):

### Checklist:

- [x] I checked all content with Grammarly
- [x] I performed a self-review of my code
- [x] I made corresponding changes to the rest of the documentation
- [x] The build passes locally
- [x] My changes generate no new warnings or errors

### CODE REVIEW

**Comment by ind1xa** [comment]
> @ind1xa DO you have a memgraph PR for this? I need to create a release note for this as well

https://github.com/memgraph/memgraph/pull/1126 @vpavicic

**Comment by vpavicic** [comment]
@ind1xa will you create a new PR for the c API?

**Comment by ind1xa** [comment]
> @ind1xa will you create a new PR for the c API?
> Did I understand correctly that the PR 1126 changed both the C and C++ API?

@vpavicic I checked and documentation for C api already exists, not sure why it wasn't implemented then

File: docs/reference-guide/query-modules/implement-custom-query-modules/api/cpp-api.md [hunk 1]
```diff
@@ -713,6 +714,14 @@ Adds a label to the node.
 void AddLabel(const std::string_view label)
 ```
 
+##### RemoveLabel
+
+Removes a label from the node.
```
Comment (vpavicic) [inline docs/reference-guide/query-modules/implement-custom-query-modules/api/cpp-api.md:719]:
```suggestion
Removes a label from a node.
```

**Review by vpavicic — APPROVED** [review]

**Pull request merged**